store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -97,15 +100,31 @@
     private static final String SCHEMA = "schema";
     private static final String PROPERTIES = "properties";
     private static final String ID = "id";
-    private final Map<String, Graph> graphs = Maps.newHashMap();
+    private static final String CACHE_SERVICE_NAME = "federatedStoreGraphs";
+    private ICacheService cacheService;
     private Set<StoreTrait> traits = new HashSet<>();
     private Set<String> customPropertiesAuths;
 
-    private static List<String> getCleanStrings(final String value) {
-        final List<String> values = Arrays.asList(StringUtils.stripAll(value.split(SCHEMA_DEL_REGEX)));
-        values.remove("");
-        values.remove(null);
-        return values;
+    /**
```

Comment:
The re-ordering of these methods, although probably correct makes reviewing this class extremely difficult.